manifest-reference.md

```diff
@@ -469,12 +471,40 @@ rev = "2f5eaba864ff630ba0c3791126a3f811b6e437f3"
 Development dependencies allow to declare *dev-dependencies* in the manifest root, which are available to all tests but not exported with the project.
 
 ## Compiler flags profiles
+### Compiler flags profiles - Toml
 Compiler flags profiles can be declared in the *profiles* table. They are organised into subtables in the following order:
 
 | Subtable | Profile name | Compiler | Operating system |
 |---|:---:|:---:|:---:|
 | Example | `debug` | `gfortran` | `linux` |
 
+- Profile name can be an arbitrary string.
+- Compiler can be a string from the following list:
+	- "gfortran"
+  - "ifort"
+  - "ifx"
+  - "pgfortran"
+  - "nvfrotran"
+  - "flang"
+  - "caf"
+  - "f95"
+  - "lfortran"
+  - "lfc"
+  - "nagfor"
+  - "crayftn"
+  - "xlf90"
+  - "ftn95"
+- Operating system can be a lowercase string from the following list:
+  - "linux"
+  - "macos"
+  - "windows"
+  - "cygwin"
+  - "solaris"
+  - "freebsd"
+  - "openbsd"
+  - "unknown"
+  - "UNKNOWN"
```

Comment:
Is this a duplicate?

Reply:
I think that it was somewhere in the command line module, but it probably does not make sense to include it. I will keep just the lower-case one.